Digital Twin/sources/VantiqPushNotification.json

```diff
@@ -0,0 +1,10 @@
+{
+  "active" : true,
+  "config" : {
+    "targetVantiqApp" : true,
+    "useAPNSDevelopmentServer" : false
+  },
+  "direction" : "SINK",
+  "name" : "VantiqPushNotification",
+  "type" : "PUSH_NOTIF"
+}
```

Comment:
Shouldn't be necessary in export or project, should get regenerated properly by virtue of needing it in a collaboration type.